src/utils/provider2agent.ts

```
@@ -216,26 +216,37 @@
     agentName: "openAIAgent",
     defaultModel: "gpt-5",
     max_tokens: 8192,
+    models: ["gpt-5", "gpt-5-nano", "gpt-5-mini", "gpt-4.1", "gpt-4.1-mini", "gpt-4.1-nano", "o3", "o3-mini", "o3-pro", "o1", "o1-pro", "gpt-4o", "gpt-4o-mini"],
```

ESLint

[error] Replace `"gpt-5",·"gpt-5-nano",·"gpt-5-mini",·"gpt-4.1",·"gpt-4.1-mini",·"gpt-4.1-nano",·"o3",·"o3-mini",·"o3-pro",·"o1",·"o1-pro",·"gpt-4o",·"gpt-4o-mini"` with `⏎······"gpt-5",⏎······"gpt-5-nano",⏎······"gpt-5-mini",⏎······"gpt-4.1",⏎······"gpt-4.1-mini",⏎······"gpt-4.1-nano",⏎······"o3",⏎······"o3-mini",⏎······"o3-pro",⏎······"o1",⏎······"o1-pro",⏎······"gpt-4o",⏎······"gpt-4o-mini",⏎····`

(prettier/prettier)